From 420df6db01107cf7ec5735a9763de719e60b0f38 Mon Sep 17 00:00:00 2001
From: Kyle Edwards <kyedwards@nvidia.com>
Date: Tue, 25 Mar 2025 12:42:19 -0400
Subject: [PATCH] Remove uninstall target

This is interfering with other projects that also have an uninstall
target.
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed7e5bc8..cdd05256 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -400,4 +400,3 @@ if(NOT _is_multi_config_generator)
 endif()
 
 configure_file(cmake/uninstall.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/proj_uninstall.cmake @ONLY)
-add_custom_target(uninstall COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/proj_uninstall.cmake)
-- 
2.34.1

